views/index.erb

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Homepage</title>
+	<style type="text/css">
+		body {
+			text-align: center; 
+			color: blue;
+		}
+		input {
+			width: 20vw;
+			font-size: 1rem !important;
+			color: green;
+			background-color: pink;
+		}
+	</style>
+</head>
+<body>
+	<h1>Boy, you must be really bored to wanna play this game. </h1>
+	<h2>Click to register below </h2>
+	<form action="/form" method="get">
```

Comment:
> Is there a reason why you chose to link to another page here? Could the registration form be on this page?
> 
> If you do decide to link to another page, we don't need to use a form here since no data is being sent to the controller - an `a href` link here would work fine.

just thought it might put people off to put a form on the first page. link is brilliant idea!, Thx